.circleci/config.yml

```diff
@@ -9,24 +9,24 @@ workflows:
     jobs:
```

Comment:
You will need to update the orb version above this to `ios: wordpress-mobile/ios@0.0.32` so it has the needed fixes.